x-pack/legacy/plugins/graph/public/angular/templates/_graph.scss

```diff
@@ -75,7 +75,7 @@
 }
 
 .gphGraph__menus, .gphGraph__bar {
```

Comment:
`.graph__menus` can be removed here

Reply:
It's still in use for the inspect panel:
<img width="1240" alt="Screenshot 2019-09-17 at 14 37 32" src="https://user-images.githubusercontent.com/1508364/65042235-bc3d6380-d958-11e9-8a8d-071ad887c065.png">

Reply:
Ahh yeah, nm then